frontend/services/entities/operating_systems.ts

```diff
@@ -92,7 +92,9 @@ const getOSVersion = ({
   teamId,
 }: IGetOsVersionOptions): Promise<IOSVersionResponse> => {
   const endpoint = endpoints.OS_VERSION(os_version_id);
-  const path = teamId ? `${endpoint}?team_id=${teamId}` : endpoint;
+  const queryString = buildQueryStringFromParams({ team_id: teamId });
+  const path =
+    typeof teamId === "undefined" ? endpoint : `${endpoint}?${queryString}`;
```

Comment:
this way seems more complex?

Reply:
Oh I see, since it may be `0` now. Would it be cleaner to keep the ternaries and use a `teamId !== undefined` condition?

Reply:
1000% ty